README.md

```diff
@@ -77,19 +77,25 @@ For adding support for other cloud providers, open an issue or propose a PR.
 
 To try Claudie you can follow these few steps or go to [Getting Started](https://docs.claudie.io/latest/getting-started/get-started-using-claudie/) section in our [documentation](docs.claudie.io).
 
-1. Download and extract manifests of the lates release from our [release page](https://github.com/berops/claudie/releases).
+1. Before you begin, please make sure you have installed [cert-manager](https://cert-manager.io/docs/installation/). 
+  
+    ```sh
+    kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.12.0/cert-manager.yaml
```

Comment:
Are we "pinning" the version of cert-manager to `v1.12.0`?

Reply:
This was the one with witch I tested the controller. I could edit this by setting the version in ENV, and add a line defining it. In the docs.claudie.io I added a line:  

*For more advanced cert-manager installation, please refer to the [official documentation](https://cert-manager.io/docs/installation/).*

WDYT?

Reply:
Sounds good to me :)